DrinkManagerWeb/Startup.cs

```diff
@@ -53,8 +67,16 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IDrinkSearchService, DrinkSearchService>();
             services.AddScoped<IFavouriteRepository, FavouriteRepository>();
             services.AddScoped<IReviewRepository, ReviewRepository>();
+            services
+                .AddRazorPages()
+                .AddViewLocalization();
+            services.AddMvc()
```

Comment:
AddMvc method was used for .Net Core 2.0-2.2 in 3.1 we should chain AddDataAnnotationsLocalization after AddControllersWithViews.

Reply:
Changed